screenpipe-server/src/migrations/20250117134500_create_ocr_text_embeddings.sql

```diff
@@ -0,0 +1,8 @@
+-- Create ocr_text_embeddings table
+CREATE TABLE IF NOT EXISTS ocr_text_embeddings (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    frame_id INTEGER NOT NULL,
+    embedding BLOB NOT NULL,  -- Store embeddings as BLOB since they're float arrays
```

Comment:
why blob instead of float array like in speaker embeddings?

can you do semantic search over this data type?

Reply:
whats the diff